docs/installation.md

```diff
@@ -71,6 +71,18 @@ sudo PIPX_HOME=/opt/pipx PIPX_BIN_DIR=/usr/local/bin pipx install PACKAGE
 # Example: $ sudo PIPX_HOME=/opt/pipx PIPX_BIN_DIR=/usr/local/bin pipx install cowsay
 ```
 
+!!! note
+
+    After version 1.2.0, the default pipx paths have been moved from `~/.local/pipx` to `$XDG_DATA_HOME/pipx`, which is typically `~/.local/share/pipx`, to ensure compatibility with the [XDG base directory specification](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html#variables).
+    
+    | Old Path               | New Path                     |
+    | ---------------------- | ---------------------------- | 
+    | `~/.local/pipx/.trash` | `$XDG_DATA_HOME/pipx/trash`  |
+    | `~/.local/pipx/shared` | `$XDG_DATA_HOME/pipx/shared` |
+    | `~/.local/pipx/venvs`  | `$XDG_DATA_HOME/pipx/venv`   |
+    | `~/.local/pipx/.cache` | `$XDG_CACHE_HOME/pipx`       |
+    | `~/.local/pipx/logs`   | `$XDG_STATE_HOME/pipx/log`   |
```

Comment:
How about linking to [this part of the platformdirs documentation](https://platformdirs.readthedocs.io/en/latest/api.html#platforms) in the above paragraph? That contains all relevant platform-specific information.

And I'd suggest replacing `~` in the table with something like `HOMEPATH`.

Reply:
`~` seems better to me, it’s more or less commonly understood as to be the home directory (even to Windows users; I think Powershell can handle this shorthand), but something like `HOMEPATH` is not universal.